### DISCUSSION TITLE
Blocks one month behind after server reboot a month ago

### DISCUSSION THREAD

**Discussion by ZebroTech** [blockscout/blockscout#8885] [General]
https://explorer1.zebrocoin.app/

blockscout latest block shows a month ago after system reboot a month ago.

**Comment by vbaranov** [comment]
I see blocks are updated in the realtime now. Please update to the new backend version [5.3.2](https://github.com/blockscout/blockscout/releases/tag/v5.3.2-beta) and consider migrating to [the new UI](https://docs.blockscout.com/for-developers/frontend-migration).

**Reply by ZebroTech** [reply]
I'm running blockscout on two servers server1 https://explorer.zebrocoin.app server2 https://explorer1.zebrocoin.app.
It synced blocks up-to-date on server 2 is (https://explorer1.zebrocoin.app) but still showing few block behind everytime currently server2 is  down again.
I tried same copy of blockscout on server1 https://explorer.zebrocoin.app and also
I tried v5.3.3 also but always blocks are two months behind.
I tried removing docker files and rebuild still blocks two months behind, two months ago both my servers were rebooted and problem started after that.
Any help will be appreciated.

**Reply by ZebroTech** [reply]
To update to the new backend version [5.3.2] and migrating to the new UI do I have to remove docker images and rebuild from scratch?

**Reply by GimluCom** [reply]
Docker will get new images if you remove the old ones. You could also fetch new images with `docker compose pull && docker image prune`.
Using `--build` when starting containers (`docker compose up -d`) is optional.
